ilastik/applets/base/appletSerializer.py

```diff
@@ -787,6 +787,19 @@ def _getValue(self, subgroup, slot):
         except AssertionError as e:
             warnings.warn('setValue() failed. message: {}'.format(e.message))
 
+class SerialFeatureNamesSlot(SerialDictSlot):
```

Comment:
Since we have also pixel features I think it would be good to rename this to something like `SerialObjectFeatureNamesSlot` (or any variation, giving the context of object features), since it is only used there, to ensure backward compatibility